src/test/java/com/google/api/codegen/testdata/csharp_gapic_snippets_library.baseline

```diff
@@ -1901,7 +1901,26 @@ namespace Google.Example.Library.V1.Snippets
             // Initialize request argument(s)
             BookName name = new BookName("[SHELF_ID]", "[BOOK_ID]");
             // Make the request
-            Operation response = await libraryServiceClient.GetBigBookAsync(name);
+            Operation<Book> response =
+                await libraryServiceClient.GetBigBookAsync(name);
+
+            // Poll until the returned long-running operation is complete
```

Comment:
Hmmm... bit of a shame we don't have a combination returning a `Task<T>` here. But we can work on that.